CHANGELOG.md

```diff
@@ -8,6 +8,68 @@ section below.
 
 _Nothing yet!_
 
+## Version 2.6.1
```

Comment:
Shouldn't this be 2.7.0 according to semantic versioning?  Since it adds to the API.

Reply:
Yes.

Reply:
Updated. Also added the new rubocop rule.